File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
def pn(i):
  if i < 2:
    return False
  x = 2
  while x * x <= i:
    if i % x == 0:
      return False
    x += 1
  return True
n = int(input())
a = list(map(int,input().split()))
cnt = 0
for i in a:
  if pn(i):
    cnt += 1
print(cnt)

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.cc
#include <iostream>
#include <string>
#include <algorithm>
#include <vector>
#include <cmath>
using namespace std;

int main() {
    int n,cnt = 0,a;
    cin >> n;
    while (n--) {
        cin >> a;
        int x = 0;
        if (a == 1)
            continue;
        else {
            for (int i = 2; i < a; i++) {
                if (a % i == 0) {
                    x = 1;
                    break;
                }
            }
            if (x) continue;
            else cnt++;
        }
    }
    cout << cnt;
    return 0;
}